core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/Command.java

```diff
@@ -41,6 +41,15 @@
    */
   String PLUGIN_ATTRIBUTE = "plugin";
 
+  /**
+   * An attribute of the command to store the original path to the file that contains the editor
```

Comment:
I think it makes sense to reference class here to make it clear which command is referenced here, devfile or workspace config

Reply:
This is in the ws config `Command` class and is therefore used for the ws config.